lib/plugins/orionPlugin.js

```diff
@@ -38,23 +38,27 @@ var sax = require('sax'),
 function translateAction(originalAction) {
     var action;
 
-    switch (originalAction.toUpperCase()) {
-        case 'APPEND':
+    switch (originalAction.toLowerCase()) {
+        case 'append':
             action = 'create';
             break;
 
-        case 'APPEND_STRICT':
+        case 'appendstrict':
```

Comment:
I'd suggest to add here the `case 'append_strict'` to keep backward compability in NGSIv1 (where APPEND_STRICT is legal and results in append_strict after toLowerCase).

Reply:
Looking the detail of the travis CI fail:

```
  2) Extract Context Broker action from request
       When a request arrives with a valid v2 JSON /op/update "APPEND_STRICT" payload to the proxy
         should add the action attribute with value "create" to the request:
```

I think that when you address the suggested change travis will be back ok.

Reply:
The case 'append_strict' to keep backward compability in NGSIv1 has been included.

Reply:
Fixed by d226823